### PR TITLE
Teams docs and logs upate

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 1]
```diff
@@ -48,6 +48,22 @@ In order to verify that the messaging endpoint is open as expected, you can surf
     - botframework.com
     - microsoftonline.com
 
+
+## Migration from Cortex XSOAR 6 to Cortex XSOAR 8 and Cortex XSIAM.
+
+### Using Cortex XSOAR or Cortex XSIAM rerouting
+ - Set the messaging endpoint in the Azure bot to be `https://ext-<CORTEXT-XSOAR-SERVER-ADDRESSS>/xsoar/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://ext-my.demisto.live/xsoar/instance/execute/teams`.
+ - Check the **long running instance** parameter in the integration instance configuration.
+ - Set the **port** paramete, it's under the connect section in the integration instance configuration.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:57]:
```suggestion
3. Set the **port** parameter. It's under the Connect section in the integration instance configuration.
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 2]
```diff
@@ -48,6 +48,22 @@ In order to verify that the messaging endpoint is open as expected, you can surf
...
+ - when using the same bot from XSOAR 6 instance, please make sure to remove the bot from the team and to add it back:
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:58]:
```suggestion
4. If using the same bot from the XSOAR 6 instance, make sure to remove the bot from the team and to add it back:
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 3]
```diff
@@ -48,6 +48,22 @@ In order to verify that the messaging endpoint is open as expected, you can surf
     - botframework.com
     - microsoftonline.com
 
+
+## Migration from Cortex XSOAR 6 to Cortex XSOAR 8 and Cortex XSIAM.
+
+### Using Cortex XSOAR or Cortex XSIAM rerouting
+ - Set the messaging endpoint in the Azure bot to be `https://ext-<CORTEXT-XSOAR-SERVER-ADDRESSS>/xsoar/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://ext-my.demisto.live/xsoar/instance/execute/teams`.
+ - Check the **long running instance** parameter in the integration instance configuration.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:56]:
```suggestion
1. Set the messaging endpoint in the Azure bot to be `https://ext-<CORTEXT-XSOAR-SERVER-ADDRESSS>/xsoar/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g., `https://ext-my.demisto.live/xsoar/instance/execute/teams`.
2. Check the **long running instance** parameter in the integration instance configuration.
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 4]
```diff
@@ -48,6 +48,22 @@ In order to verify that the messaging endpoint is open as expected, you can surf
...
+ - Set the **port** paramete, it's under the connect section in the integration instance configuration.
+ - when using the same bot from XSOAR 6 instance, please make sure to remove the bot from the team and to add it back:
+   - Go to Microsoft Teams app.
+   - Go to your team, and click on the three dots next to the name
+   - Go to `manage team` -> `apps`.
+   - Find your bot, and click on the three dots in the same row.
+   - choose `remove`.
+   - Add the bot again to the team.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:64]:
```suggestion
   a. Go to the Microsoft Teams app.
   b. Go to your team, and click the three dots next to the name.
   c. Go to **manage team** > **apps**.
   d. Find your bot, and click the three dots in the same row.
   e. Click **remove**.
   f. Add the bot to the team.
```

File: Packs/MicrosoftTeams/ReleaseNotes/1_4_56.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Microsoft Teams
+
+- Documentation and metadata improvements.
+- Note: added documentation to the process of migrating from Cortex XSOAR 6 to Cortex XSOAR 8 and Cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_4_56.md:7]:
```suggestion
- Added documentation for the process of migrating from Cortex XSOAR 6 to Cortex XSOAR 8 and Cortex XSIAM.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @michal-dagan Doc review completed.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**